utils/utils.jsx

```diff
@@ -1345,22 +1297,15 @@ export function sortUsersByStatusAndDisplayName(users, statusesByUserId) {
             return UserStatusesWeight[aStatus] - UserStatusesWeight[bStatus];
         }
 
-        const aName = getDisplayNameByUser(a);
-        const bName = getDisplayNameByUser(b);
+        const aName = getDisplayNameByUser(state, a);
+        const bName = getDisplayNameByUser(state, b);
```

Comment:
These could use `displayUsername` from `mattermost-redux` so that instead of passing the whole state tree around as much, we could just pass the `teammateNameDisplay` setting instead